src/redux/thunks/previewCluster.ts

```diff
@@ -66,7 +72,7 @@ const previewClusterHandler = async (context: string, thunkAPI: any) => {
       r => r.kind === 'CustomResourceDefinition'
     );
     if (customResourceDefinitions.length > 0) {
-      const customResourceObjects = await loadCustomResourceObjects(kc, customResourceDefinitions);
+      const customResourceObjects = await loadCustomResourceObjects(kc, customResourceDefinitions, clusterAccess.namespace as string);
```

Comment:
similar question about `clusterAccess.namespace` if it could be `undefined` ?

Reply:
Missed this one, if namespace is undefined it will not have an impact since custom resources are not namespaces, will change the code for this

Reply:
I recently added an isNamespaced property to the kind handlers (#1438) - so you can check if a specific resource kind should have a namespace or not... could that help?